docs/pages/world/account-delegation.mdx

```diff
@@ -17,6 +19,20 @@ This feature enables multiple use cases.
   This is conceptually similar to how ERC20's [`approve`/`transferFrom`](https://eips.ethereum.org/EIPS/eip-20#transferfrom) enables atomic swaps by allowing contracts to withdraw from a user's balance and then deposit a different asset in a single transaction.
   For example, an agent could exchange two in-game assets atomically if the two sides of the trade give it the necessary permission.
 
+<details>
+
+<summary>Delegation and Account Abstraction</summary>
+
+Both [ERC-4337](https://www.erc4337.io/) and delegation allow an account to act on behalf of another account.
+However, while there is some overlap, the identity of the delegatee, the account acting on behalf of the original wallet, is different.
```

Comment:
I'm not sure the first sentence is correct - I woudn't say ERC-4337 allows an account to act on behalf of another account. It's more that it enables different types of _signatures_ to be used to send transactions. With an EOA you have to use an Ethereum private key to sign a transaction and send it to the mempool, 4337 allows you to use different types of authentication, verify it onchain, and then "execute" the transaction from the smart account contract.

The overlap is this: you _can_ use Ethereum signatures to signer `UserOps` in 4337, in which case you're signing from one account and executing from another (the smart account contract). In MUD we use a session wallet to sign transactions on behalf of another account, so that's kind of similar, but still pretty different. You kind of describe this below but I think the difference and how they can work together it not quite clear yet. Let me know if you want me to provide more context!

Reply:
Can you check if the new explanation is better? If not, I might need more context.